crowbar_framework/config/routes.rb

```diff
@@ -238,7 +238,7 @@
     via: [:get, :post, :put, :patch, :delete]
 
   namespace :api,
-    constraints: ApiConstraint.new(version: 2.0) do
+    constraints: ApiConstraint.new(major: 2, minor: 0) do
```

Comment:
BTW, are you tagging your `v1` APIs? Should they be a part of `v2` API namespace as well?

Maybe `ApiConstraint.new(1=>4, 2=>0)` will be a better API? It will allow you to tag an endpoint/namespace as a part of several major versions...